exporter/signalfxexporter/internal/translation/constants.go

```diff
@@ -55,345 +55,409 @@ translation_rules:
     user: true
     wait: true
 - action: aggregate_metric
-  metric_name: system.cpu.usage
+  metric_name: sf_temp.system.cpu.usage
   aggregation_method: sum
   without_dimensions:
   - state
 - action: copy_metrics
   mapping:
-    system.cpu.delta: system.cpu.total
+    sf_temp.system.cpu.delta: sf_temp.system.cpu.total
 - action: aggregate_metric
-  metric_name: system.cpu.total
+  metric_name: sf_temp.system.cpu.total
   aggregation_method: sum
   without_dimensions:
   - state
 - action: calculate_new_metric
   metric_name: cpu.utilization_per_core
-  operand1_metric: system.cpu.usage
-  operand2_metric: system.cpu.total
+  operand1_metric: sf_temp.system.cpu.usage
+  operand2_metric: sf_temp.system.cpu.total
   operator: /
 - action: copy_metrics
   mapping:
-    cpu.utilization_per_core: cpu.utilization
+    cpu.utilization_per_core: sf_temp.cpu.utilization
 - action: aggregate_metric
-  metric_name: cpu.utilization
+  metric_name: sf_temp.cpu.utilization
   aggregation_method: avg
   without_dimensions:
   - cpu
+- action: multiply_float
+  scale_factors_float:
+    sf_temp.cpu.utilization: 100
+- action: rename_metrics
+  mapping:
+    sf_temp.cpu.utilization: cpu.utilization
```

Comment:
I agree on avoiding usage of metrics like `system.cpu.total` which are discarded at the end, by don't get why do we need to introduce temporary `sf_temp.cpu.utilization` if it replaces `cpu.utilization` eventually anyway? 

All these operations are pretty performance sensitive, so I would avoid introducing any extra steps like this that doesn't seem to be necessary.

Reply:
>by don't get why do we need to introduce temporary sf_temp.cpu.utilization if it replaces cpu.utilization eventually anyway?

Not sure I follow your question.  Where does `sf_temp.cpu.utilization` replace `cpu.utilization`?

If we don't only transform/scale the intermediary metrics incoming metrics with the same name will also have their values scaled.  If a collector is used as a gateway w/ sfx exporter and default translation rules all received `cpu.utilization` metrics are multiplied by 100 because of the current:

```
- action: multiply_float
  scale_factors_float:
    memory.utilization: 100
    cpu.utilization: 100
```

I do think the generated `cpu.utilization_per_core` should also be `sf_temp` prefixed to ensure we're similarly not adjusting existing ones needlessly.

Reply:
> Not sure I follow your question. Where does sf_temp.cpu.utilization replace cpu.utilization?

Sorry, just bad phrasing. I mean that `sf_temp.cpu.utilization` merged into `cpu.utilization` anyway, so why do we need `sf_temp.cpu.utilization` and can't use `cpu.utilization` instead?

> If a collector is used as a gateway w/ sfx exporter and default translation rules all received cpu.utilization metrics are multiplied by 100 because of the current.

Oh I didn't know that this PR is intended to solve this gateway issue. I don't think it completely solves it. All the native otel metrics are still being passed to the gateway. So if the gateway has translations enabled, all the same computations will be performed, and even with `sf_temp.cpu.utilization`, we will end up with at least double-publishing `cpu.utilization` metrics, right? I assume some other issues might be involved. If we want to solve it here, we would need to drop existing `cpu.utilization` first.

As another option, I think it would be better if we just make sure that translations just disabled in the gateway default config. How does it sound? Actually I thought I did it in the helm chart initially, but don't see this anymore, maybe due to some config interface changes.

Reply:
Oh the chart is good. we use otlp to send to the gateway, so no translations on the agent. Same recommendation documented for linux/windows setup

Reply:
The gateway was an example of the general issue, which is there is a set of metric names whose values will be unexpectedly changed if using the sfx exporter with default translation rules.  I think this is poor ux overall and we shouldn't allow it in the translation rules themselves.  double publishing is a good concern here, but I wonder if we should be exporting constituent metrics like `system.cpu.time` to begin with (would avoid this case).

If we're using agent otlp exporter to the gateway and need the translation rules for bw compat metrics w/ sfx exporter then should we be using defaults in gateway?  If not where do they come from?

Reply:
> I wonder if we should be exporting constituent metrics like system.cpu.time to begin with (would avoid this case).

Looks like `system.cpu.time` not being exported already, because it processed by `split_metric`, so we should be good. But I see other cases where we are getting double publishing with and without temporary metrics , for example calculation of extra `system.network.packets.total` and `system.network.io.total` metrics

> If we're using agent otlp exporter to the gateway and need the translation rules for bw compat metrics w/ sfx exporter then should we be using defaults in gateway? If not where do they come from?

This is what configured in the chart, yes. translations are done only on the gateway